### ISSUE TITLE
r-universe: confirm build error is fixed as part of next release

### ISSUE THREAD

**Issue by mccalluc** [opendp/opendp#1643]
Right now on r-universe we show two packages:

https://opendp.r-universe.dev/builds:
- 2024-05-23	[opendp-dev/opendp](https://opendp-dev.r-universe.dev/opendp)	[0.9.2](https://github.com/opendp/opendp/commit/fee80bbe75d96e2c405c86cf2f1f9268941cde3f)	[Stefano Iacus](https://siacus.r-universe.dev/)						
- 2024-02-08	[opendp](https://opendp.r-universe.dev/opendp)[(build failure)](https://github.com/r-universe/opendp/actions/runs/9012563114)	[0.9.2](https://github.com/opendp/opendp/commit/c5329d6430f0ff51dd599361a25af890fae84f65)	[Stefano Iacus](https://siacus.r-universe.dev/)

Before the next release, can we make sure that https://opendp.r-universe.dev/opendp# reports a successful build? 

It turns out that r-universe is using github under the hood: The build failure for 0.9.2 is [here](https://github.com/r-universe/opendp/actions/runs/9012563114/job/24862382967#step:3:486)
```
  convert.c: In function ‘error_unknown_type’:
  convert.c:22:5: error: format not a string literal and no format arguments [-Werror=format-security]
     22 |     error(msg);
        |     ^~~~~
  convert.c: In function ‘extract_error’:
  convert.c:74:9: error: format not a string literal and no format arguments [-Werror=format-security]
     74 |         error(msg);
        |         ^~~~~
  convert.c:80:9: error: format not a string literal and no format arguments [-Werror=format-security]
     80 |         error(msg_backtrace);
        |         ^~~~~
  cc1: some warnings being treated as errors
```

(But the nightly build is successful, so hopefully this will just go away with the next release.)

**Comment by mavelli** [comment]
`> install.packages("opendp", repos = "https://opendp.r-universe.dev")
trying URL 'https://opendp.r-universe.dev/src/contrib/opendp_0.9.2.tar.gz'`
still errors. cannot build and use opendp in r:
`gcc -I"/usr/include/R/" -DNDEBUG   -I/usr/local/include    -fpic  -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -fexceptions         -Wp,-D_FORTIFY_SOURCE=3 -Wformat -Werror=format-security         -fstack-clash-protection -fcf-protection         -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -g -ffile-prefix-map=/build/r/src=/usr/src/debug/r -flto=auto -ffat-lto-objects  -c transformations.c -o transformations.o
* extract opendp and vendor files
convert.c: In function ‘error_unknown_type’:
convert.c:22:5: error: format not a string literal and no format arguments [-Werror=format-security]
   22 |     error(msg);
      |     ^~~~~
convert.c: In function ‘extract_error’:
convert.c:74:9: error: format not a string literal and no format arguments [-Werror=format-security]
   74 |         error(msg);
      |         ^~~~~
convert.c:80:9: error: format not a string literal and no format arguments [-Werror=format-security]
   80 |         error(msg_backtrace);
      |         ^~~~~
cc1: some warnings being treated as errors
make: *** [/usr/lib64/R/etc/Makeconf:195: convert.o] Error 1`

**Comment by Shoeboxam** [comment]
This error came up in #1536, and will be fixed upon our next release (0.10), which we are preparing now.

**Comment by mccalluc** [comment]
https://opendp.r-universe.dev/builds looks good:

> 2024-06-20	[opendp](https://opendp.r-universe.dev/opendp)	[0.10.0](https://github.com/opendp/opendp/commit/9a674cd5ee26dbb9b62ef987435b5cadbf936d21)

haven't tried locally, but the original issue was just the build error in r-universe, so I think this is sufficient to close. File a new issue if there's a local problem.
